From randy.dunlap@oracle.com  Thu Nov 19 11:19:12 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 12 Nov 2009 15:46:01 -0800
Subject: Staging: line6: fix printk formats
To: Stephen Rothwell <sfr@canb.auug.org.au>, gregkh@suse.de
Message-ID: <20091112154601.97a4e2ab.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix printk format warnings in line6/pod.c; sizeof() is of type
size_t, so use %zu.

drivers/staging/line6/pod.c:581: warning: format '%d' expects type 'int', but argument 4 has type 'long unsigned int'
drivers/staging/line6/pod.c:693: warning: format '%d' expects type 'int', but argument 4 has type 'long unsigned int'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/line6/pod.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/line6/pod.c
+++ b/drivers/staging/line6/pod.c
@@ -579,7 +579,7 @@ static ssize_t pod_set_dump(struct devic
 
 	if (count != sizeof(pod->prog_data)) {
 		dev_err(pod->line6.ifcdev,
-			"data block must be exactly %d bytes\n",
+			"data block must be exactly %zu bytes\n",
 			sizeof(pod->prog_data));
 		return -EINVAL;
 	}
@@ -691,7 +691,7 @@ static ssize_t pod_set_dump_buf(struct d
 
 	if (count != sizeof(pod->prog_data)) {
 		dev_err(pod->line6.ifcdev,
-						"data block must be exactly %d bytes\n",
+						"data block must be exactly %zu bytes\n",
 						sizeof(pod->prog_data));
 		return -EINVAL;
 	}
